setup.cfg

```diff
@@ -62,3 +62,4 @@ dev =
      sphinx-rtd-theme>=1.0.0
      testfixtures>=6.18.3
 
+#
```

Comment:
Ah, I see, a trigger!  Maybe remove this prior to actual PR so it isn't hiding in there, but I see a broader problem relying on changes to setup.cfg being our trigger anyway - I'll talk about that in the broader PR